theano/gpuarray/c_code/dnn_sptf_desc.c

```diff
@@ -0,0 +1,46 @@
+#section support_code_apply
+
+int APPLY_SPECIFIC(dnn_sptf_desc)(PyArrayObject * dims,
+                                  cudnnSpatialTransformerDescriptor_t * desc,
+                                  PARAMS_TYPE * params)
+{
+    cudnnStatus_t err;
+
+    const int nimages = *((int *) PyArray_GETPTR1(dims, 0));
```

Comment:
See my comments below (in `dnn.py`), but it is quite dangerous to cast something to an int without some checkings.